main.py

```diff
@@ -79,7 +79,30 @@ async def on_message(message: Message):
             await message.channel.send("You need to use !list first, sugar.")
             return
 
-        await command_play(message)
+        # split on spaces
+        # TODO: Possibly parse arguments before casing on command
+        command_args = message.content.split(" ")
```

Comment:
To help make reasoning about this list easier, I may remove the command itself from the list, so `command_args` consists of only the index.

```suggestion
        command_args = message.content.split()[1:]
```

Though you may want to split that step out into a separate line for better readability.